src/models/bookModel.ts

```diff
@@ -0,0 +1,6 @@
+export interface Book {
```

Comment:
To differentiate between model types we can add a keyword to specify where they come from (DbBook for database models, ApiBook for api models, etc). This is usually done when we have to map db models to api models before returning them. So it's not a real issue here but it's a good practice.